packages/plugin-browser-device/device.js

```diff
@@ -3,13 +3,17 @@ const assign = require('@bugsnag/core/lib/es-utils/assign')
 /*
  * Automatically detects browser device details
  */
-module.exports = (nav = navigator) => ({
+module.exports = (nav = navigator, _screen = window.screen) => ({
   load: (client) => {
     const device = {
       locale: nav.browserLanguage || nav.systemLanguage || nav.userLanguage || nav.language,
       userAgent: nav.userAgent
     }
 
+    if (_screen && _screen.orientation && _screen.orientation.type) {
+      device.orientation = _screen.orientation.type
```

Comment:
The [`window.screen.orientation` API](https://developer.mozilla.org/en-US/docs/Web/API/Screen/orientation) isn't supported by all of the browsers in our support matrix, and for some it requires a vendor prefix (see the example on MDN).

It would be good to fall back to the result of this expression:

`document.documentElement.clientWidth > document.documentElement.clientHeight ? 'landscape' : 'portrait'`

which I tested in IE8

It would also be good to add a maze runner test for this. Asserting that the value is one of `landscape` `portrait` or one of the [`ScreenOrientation.type` values](https://developer.mozilla.org/en-US/docs/Web/API/ScreenOrientation/type).